src/components/Container/Container.js

```diff
@@ -0,0 +1,9 @@
+import styles from './Container.module.scss';
+
+const Container = props => {
+    return (   
+      <div className={styles.container}>{props.children}</div>   
+    );
+  };
```

Comment:
Możesz zrobić to nawet krócej stosując destrukturyzację propsów:
```suggestion
const Container = ({children}) => {
    return (   
      <div className={styles.container}>{children}</div>   
    );
  };
```